ophyd/controls/positioner.py

```diff
@@ -425,8 +421,6 @@ def __init__(self, setpoint, readback=None,
                                         name=self.name))
 
             self._readback.subscribe(self._pos_changed)
-
-            self._set_position(self._readback.value)
```

Comment:
My concern is more for the case where the `EpicsMotor` has just been created but has yet to connect. We got around that previously by blocking on creation. This at least ensured the initial state.

A (more complex) solution for `ensure_connection` might check to see if the initial connection attempt has finished yet. If not, it blocks. In the case of a random disconnection, it raises immediately.